### DISCUSSION TITLE
Calendar With Different Color

### DISCUSSION THREAD

**Discussion by MrCasquette** [directus/directus#17262] [Feature Requests]
It would be nice to have the possibility to adjust the color of an event by switching from one source or resource to another, as indicated in the full calendar documentation. see : [Full Calendar Resource doc](https://fullcalendar.io/docs/resource-data)

I mean if I want to create a schedule for my company having for example 4 employees. It would be interesting if each employee had a different color to differentiate them all. In list view or even in week view. And there you have it, a great company schedule.

@rijkvanzanten it's up to you 😄 I'm sure you can do it

**Comment by paescuj** [comment]
(Ref https://github.com/directus/directus/discussions/6024)

**Reply by MrCasquette** [reply]
Thanks @paescuj   @rijkvanzanten asked me to write this request maybe I mispoke in my previous post. :)

**Reply by paescuj** [reply]
Sure, I've just added the reference to keep track of this thread because there's a similar request in it 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by demic-dev** [comment]
Hello! Since this feature is also useful to my company in a project I'm working on, I'm considering forking the project to work on this independently. However, would it be ok if I would submit a PR for this request as well? I was thinking of adding a color field to each object and, through the right column selecting the color field (like with start and end time).

What do you think?

I've read the contributions terms and I know that this request is closed, but since I would implement this in my self hosted project, I think it would be nice to contribute here too :)
